api/src/poem_backend/runtime.rs

```diff
@@ -144,6 +145,12 @@ pub fn attach_poem_to_runtime(
             .nest("/", api_service)
             .at("/spec.json", spec_json)
             .at("/spec.yaml", spec_yaml)
+            // TODO: We add this manually outside of the OpenAPI spec for now.
+            // https://github.com/poem-web/poem/issues/364
+            .at(
+                "/set_failpoint",
+                poem::get(set_failpoints::set_failpoint_poem).data(context.clone()),
+            )
```

Comment:
Yeah, I don't think this needs to be in the spec anyways.  It's not a public facing feature, but instead a local feature.  This should be turned off by default btw.

Reply:
The endpoint is there but fails if you call it:
```
{"message":"Failpoints are not enabled at a feature level","error_code":null,"aptos_ledger_version":null}
```

It was easiest to do it this way for now.